src/solver/SolverLayout.tsx

```diff
@@ -2,7 +2,7 @@ import dagre from '@dagrejs/dagre';
 import {
   Background,
   BackgroundVariant,
-  ConnectionLineType,
+  ConnectionLineType, ControlButton,
```

Comment:
May you run `npm run format` after changes? We are using prettier it should be applied directly in many editor, not sure about IntelliJ

Reply:
Check, my bad.